streamlit_web.py

```diff
@@ -181,7 +191,22 @@ def import_data(data_retreive_method, paths):
                 json_lst.append(ju.clean_report(response))
         except (EnvironmentError, Exception) as err:
             st.sidebar.error(err)
-            landing_src()
+            with open("README.md") as readme_file:
+                st.markdown(readme_file.read())
+    else:
+        try:
+            if len(paths) < 2:
+                st.sidebar.warning("Please select more than one file!")
+            else:
+                for path in paths:
+                    stringio = StringIO(path.getvalue().decode("utf-8"))
+                    individual_dict = md.md_parser(stringio.read(), True)
+                    main_md_dict = md.merge_dict(main_md_dict, individual_dict)
+                json_lst.append(main_md_dict)
+        except FileNotFoundError as err:
+            st.sidebar.text(err)
```

Comment:
This seems to be doing its own standalone thing. Maybe a better idea to encapsulate this in a separate function.